plugin/hotswap-agent-spring-plugin/pom.xml

```diff
@@ -28,6 +28,7 @@
             <groupId>org.springframework</groupId>
             <artifactId>spring-web</artifactId>
             <version>${org.springframework.version}</version>
+            <scope>provided</scope>
```

Comment:
It's interfering with debugging if the actual spring version is not ${org.springframework.version} and scope is not set to provided.